index.html

```diff
@@ -54,8 +54,206 @@ <h2>LET'S CONNECT</h2>
+        <div class="heading">
+        <h3>Multi-post stories</h3>
+        </div>
+        <div class="skills">
+          <ul>
+            <li>
+              <span class="canopy">Canopy</span>
+            </li>
+            <li>
+              <div class="div"></div>
+            </li>
+            <span>Full Stack Dev</span>
+            <li>
+              <div class="div"></div>
+            </li>
+            <span>2015</span>
+          </ul>
+        </div>
+        <p class="heading2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="tech">
+          <ul>
+            <li><span>HTML</span> </li>
+            <li><span>CSS</span> </li>
+            <li><span>Javascript</span> </li>
+          </ul>
+        </div>
+        <button type="submit" class="btn-a ">See Projects</button>
+      </div>
+    </article>
 
+    <article class="card-container">
+      <div class="gb-img">
+        <img src="img/Snapshoot Portfoliolast.png">
+      </div>
+      <div class="contents">
+        <div class="heading">
+          <h3>Tonic</h3>
+        </div>
+        <div class="skills">
+          <ul>
+            <li>
+              <span class="canopy">Canopy</span>
+            </li>
+            <li>
+              <div class="div"></div>
+            </li>
+            <span>Full Stack Dev</span>
+            <li>
+              <div class="div"></div>
+            </li>
+            <span>2015</span>
+          </ul>
+        </div>
+        <p class="heading2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="tech">
+          <ul>
+            <li><span>HTML</span> </li>
+            <li><span>CSS</span> </li>
+            <li><span>Javascript</span> </li>
+          </ul>
+        </div>
+        <button type="submit" class="btn-a ">See Projects</button>
+      </div>
+    </article>
 
+    <!-- card 4 -->
 
+    <article class="card-container">
+      <div class="gb-img">
+       <img src="img/Snapshoot Portfolio2.png">
+      </div>
+      <div class="contents">
+        <div class="heading">
+          <h3>Multi-post stories</h3>
+        </div>
+        <div class="skills">
+          <ul>
+            <li>
+              <span class="canopy">Canopy</span>
+            </li>
+            <li>
+              <div class="div"></div>
+            </li>
+            <span>Full Stack Dev</span>
+            <li>
+              <div class="div"></div>
+            </li>
+            <span>2015</span>
+          </ul>
+        </div>
+        <p class="heading2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="tech">
+          <ul>
+            <li><span>HTML</span> </li>
+            <li><span>CSS</span> </li>
+            <li><span>Javascript</span> </li>
+          </ul>
+        </div>
+        <button type="submit" class="btn-a ">See Projects</button>
```

Comment:
- [ ] Well done :clap: you did almost all changes given by the previous reviewer :point_up_2: but I noticed that you did fix your button style and hover, active etc effects so kindly fix this issue :+1: check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face:

| Your Design | Figma Design|
|--------|--------|
| ![Screenshot from 2023-06-07 16-27-52](https://github.com/Isaac-ej/Portfolio_Website/assets/45226848/c553c911-2de8-41e2-94b6-ee78e7b90eae) | ![Screenshot from 2023-06-07 16-30-29](https://github.com/Isaac-ej/Portfolio_Website/assets/45226848/ce203af2-319e-43c3-92d6-7a24182c7624) |  

_**The following :point_down: are the button effects so kindly apply these all effects given below**_

![Screenshot from 2023-02-17 17-31-04](https://github.com/Isaac-ej/Portfolio_Website/assets/45226848/d0be23b3-fd48-4002-8b81-5aa587fa1238)